### ISSUE TITLE
Running qunit tests from the command line produces no output.

### ISSUE THREAD

**Issue by klenwell** [modeset/teaspoon#390]
Following instructions in README and [wiki](https://github.com/modeset/teaspoon/wiki/Using-QUnit). Tests run in browser with qunit as expected. 

When I set up teaspoon with jasmine and run tests from the command line, they run ok. I see the following output:

    $ xvfb-run bundle exec rake teaspoon DRIVER=capybara_webkit suite=default
    Starting the Teaspoon server...
    Teaspoon running default suite at http://127.0.0.1:34088/teaspoon/default
    ..
    
    Finished in 0.00800 seconds
    2 examples, 0 failures
    $

But when I run the tests from the command line with qunit, I get nothing:

    $ xvfb-run bundle exec rake teaspoon DRIVER=capybara_webkit suite=default
    Starting the Teaspoon server...
    Teaspoon running default suite at http://127.0.0.1:47364/teaspoon/default
    
    $

Note: I set up qunit and jasmine in separate branches. In my qunit branch, my test file is located at `test/javascripts/spike_test.js`.

Have I missed something?

**Comment by jejacks0n** [comment]
If make sure your paths are correct in the confit. You can generate the qunit setup using the generator, which is confirmed as working.

---
Jeremy Jackson

**Comment by klenwell** [comment]
@jejacks0n Yes, I generated the setup for both qunit and jasmine with the generator.

> If make sure your paths are correct in the confit.

Do you mean that I should double check my paths in the `teaspoon_env.rb` file? I used the defaults, which look sane.

**Comment by jejacks0n** [comment]
Check the project specs then? We can run them from an empty rails app, so you should be able to as well.

---
Jeremy Jackson

**Comment by klenwell** [comment]
I'd expect the project specs to be consistent between the qunit and jasmine branches I set up. So I don't think that would explain the issue. Unless I'm misunderstanding what is mean by term project specs.

In any event, I'm ready to accept that it's something peculiar to my current project setup. I just wanted to raise the issue here in case there was something obvious that I had missed and as a record in the event somebody else experience the same issue.

**Comment by jejacks0n** [comment]
Term = repo -- sorry, on mobile.

---
Jeremy Jackson

**Comment by mikepack** [comment]
I see you're using capybara webkit and you may have run into this now-resolved issue: #403. Could you point your Gemfile at teaspoon master and see if it's fixed?

```ruby
gem 'teaspoon', github: 'modeset/teaspoon'
gem 'teaspoon-qunit'
gem 'capybara-webkit'
```

**Comment by mikepack** [comment]
Closing but will reopen if #403 does not fix the issue.
